UI/data/locale/en-US.ini

```diff
@@ -516,7 +516,7 @@ Basic.Main.AddSourceHelp.Text="You need to have at least 1 scene to add a source
 
 # basic mode main window
 Basic.Main.Scenes="Scenes"
-Basic.Main.Sources="Sources"
+Basic.Main.Sources="Items"
```

Comment:
I think the term "Source" is far too ingrained in people's mind that changing the title of the Sources box would be far too confusing for people, not to mention invalidate every OBS tutorial ever.